src/hipscat_import/pipeline_resume_plan.py

```diff
@@ -220,3 +207,22 @@ def get_pixel_cache_directory(cache_path, pixel: HealpixPixel):
     return file_io.append_paths_to_pointer(
         cache_path, f"order_{pixel.order}", f"dir_{pixel.dir}", f"pixel_{pixel.pixel}"
     )
+
+
+def print_task_failure(custom_message, exception):
+    """Use dask's distributed print feature to print the exception message to the task's logs
+    and to the controller job's logs.
+
+    Optionally print the worker address if a worker is found.
+
+    Args:
+        custom_message (str): some custom message for the task that might help with debugging
+        exception (Exception): error raised in execution of the task.
+    """
+    dask_print(custom_message)
+    try:
+        dask_print("  worker address:", get_worker().address)
+    except Exception:  # pylint: disable=broad-exception-caught
+        pass
+    dask_print(exception)
+    raise exception
```

Comment:
I think we don't need this instruction here because the exception is already being raised in each task's try catch

Reply:
Agreed! I thought I had even removed that line.

Thanks for the catch.